CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## [Unreleased]
+
+### Added
+- Reload Webpacker configuration and manifest when `caching` config is set to false.
+
+```rb
+# config/environments/production.rb
+config.webpacker.caching = true
+
+# config/environments/staging.rb
+config.webpacker.caching = false
+```
+
+- Add `exist?` helper method to check if configuration or manifest data exist.
+
+```rb
+Webpacker::Manifest.exist? # => false
+Webpacker::Configuration.exist? # => true
+```
+
+- Add `reloadable?` helper method to check if configuration or manifest is reloaded.
```

Comment:
Nowhere at the moment, just added it as an alias to `Webpacker.caching` in case someone wants to use this instead. Probably more readable?